frontend/core/blocks/document-extraction/specs.json

```diff
@@ -0,0 +1,69 @@
+{
+  "information": {
+    "id": "ds4sd-dociling",
+    "name": "Ds4sd Dociling",
+    "description": "Block to extract data from the files in markdown format",
```

Comment:
Please add a more comprehensive description. E.g. what is it his block used for, what files does it take, what files does it produce?